bioindex/lib/schema.py

```diff
@@ -165,15 +166,15 @@ def sql_filters(self):
         Builds the query string from the index columns that can be used in a
         SQL execute statement.
         """
-        tests = 'AND'.join(map(lambda k: f'`{k}`=%s ', self.key_columns))
+        tests = 'AND'.join(map(lambda k: f'`{k}`=:{k.replace("|", "_")} ', self.key_columns))
```

Comment:
There are restrictions around named parameters for sql alchemy and apparently pipes aren't allowed.